infrastructure/images/schema-registry/Makefile

```diff
@@ -0,0 +1,10 @@
+build:
+	docker build -t schema-registry . --build-arg SCHEMA_REGISTRY_VERSION=${SCHEMA_REGISTRY_VERSION}
```

Comment:
so the current build system assumes you have the corresponding egg file in the current directory. I did this for simplicity short-term but we should probably improve over time